src/core/config/config.go

```diff
@@ -225,6 +225,11 @@ func TokenExpiration() (int, error) {
 	return cfgMgr.Get(common.TokenExpiration).GetInt(), nil
 }
 
+// RobotTokenExpiration returns the token expiration time of robot account (in day)
+func RobotTokenExpiration() int {
```

Comment:
What if user want to set it to hours?  
To be consistent with the token by Harbor the unit should be minute?